### ISSUE TITLE
Spam in comments is visible on https://planet.sympy.org/index.html

### ISSUE THREAD

**Issue by cknoll** [friyaz/friyaz.github.io#20]
I want to recommend https://planet.sympy.org/index.html in a book that I am writing. However, currently the startpage of that site is dominated by spam comments. If I tracked it down correctly these comments are imported via this article:

https://friyaz.github.io/blog/GSoC-Final_Report

It would be nice to remove them. This would give planet.sympy.org and of course also your blog a much more professional appearance. 

(Feel free to correct me if my observation is wrong and your blog is not the source.)

PS: Thank you for your work on SymPy! :+1:

**Comment by faisal-deepsource** [comment]
Hey @cknoll, I am very sorry for this. I was aware of the spam comments. But I did not notice that it affected the sympy website; otherwise, I would have fixed it earlier. 

I have removed all the spam comments. I have disabled comments and will enable them back once I add reCaptcha. These changes have been reflected on [planet sympy](https://planet.sympy.org/index.html). Thanks for pointing this out. It was very irresponsible on my part. Once again, I am sorry.

Let me know when you finish the book. I will love to read it if it is related to symbolic maths.

**Comment by friyaz** [comment]
Hey @cknoll, I am very sorry for this. I was aware of the spam comments. But I did not notice that it affected the sympy website; otherwise, I would have fixed it earlier.

I have removed all the spam comments. I have disabled comments and will enable them back once I add reCaptcha. These changes have been reflected on [planet sympy](https://planet.sympy.org/index.html). Thanks for pointing this out. It was very irresponsible on my part. Once again, I am sorry.

Let me know when you finish the book. I will love to read it if it is related to symbolic maths.
